docs/getting-started/server/guide.md

```diff
@@ -272,9 +272,15 @@ Navigate to the `dev` folder in your server repo and perform the following comma
 
 ## Install Licensing Certificate
 
-To run your local server environment as a licensed instance, you will need to download the
-`Licensing Certificate - Dev` from the shared Engineering collection and install it. This can be
-done by double-clicking on the downloaded certificate.
+To run your local server environment as a licensed instance:
+
+1. Go to Bitwarden's shared Development collection. (If you don't have access to this collection,
```

Comment:
The Engineering collection is now called Development. As for the note about contacting IT if necessary, I've moved this note here because this is the first place in the setup guide that mentions the Development Collection.

Reply:
Makes sense!